src/base/bittorrent/session.cpp

```diff
@@ -612,6 +613,17 @@ void Session::setAppendExtensionEnabled(const bool enabled)
     }
 }
 
+uint Session::stop_tracker_timeout() const
+{
+    return m_stop_tracker_timeout;
+}
+void Session::setstop_tracker_timeout(const uint value)
```

Comment:
Missing space, and function name should be in camelCase for consistency with the others.